### ISSUE TITLE
Choice UI cannot see the Choice tests

### ISSUE THREAD

**Issue by markomandic** [Sleashe/choice#1]
I followed the docker instructions for setting up choice and choice ui. I have successfully created choice tests and I'm able to use the choice-client to take decisions. However the choice ui is unable to see the tests that have been created, any ideas on what could be wrong?

Querying choice db:
request('http://0.0.0.0:80/choice', '{tests{id name}}').then((data) => console.log(data))
Promise {
  <pending>,
  domain:
   Domain {
     domain: null,
     _events:
      [Object: null prototype] {
        removeListener: [Function: updateExceptionCapture],
        newListener: [Function: updateExceptionCapture],
        error: [Function: debugDomainError] },
     _eventsCount: 3,
     _maxListeners: undefined,
     members: [] } }
> { tests:
   [ { id: '5f088d996611ea00017630ed', name: 'test-name' },
     { id: '5f088ee36611ea00017630ff', name: 'test-name2' },
     { id: '5f08974b6611ea0001763106', name: 'test-name3' },
     { id: '5f08b4456611ea000176310b', name: 'test-name4' } ] }

Choice UI: No test have been created yet :(

**Comment by Sleashe** [comment]
Hi @markomandic. This is an old lib I should remove... I did this project a few years ago and it's not updated anymore....
Dependencies are mainly outdated too...

I'm sorry :/

**Comment by markomandic** [comment]
I see, thank you for the quick reply :) Everything besides the Choice UI is working fine, so I'll try to see what could be wrong

**Comment by Sleashe** [comment]
Yeah feel free, to update, fix, and PR, it would be awesome thank u !
